homeassistant/components/squeezebox/media_player.py

```diff
@@ -220,11 +220,26 @@ def __init__(self, player: Player, server: Server) -> None:
         self._query_result: bool | dict = {}
         self._remove_dispatcher: Callable | None = None
         self._attr_unique_id = format_mac(player.player_id)
+        if player.model == "SqueezeLite":
+            _manufacturer = "https://github.com/ralph-irving/squeezelite"
+        elif (
+            "Squeezebox" in player.model
+            or "Transporter" in player.model
+            or "Slim" in player.model
+        ) or "Transporter" in player.model:
+            _manufacturer = "Logitech"
+        elif "SqueezePlay" in player.model:
+            _manufacturer = "https://github.com/ralph-irving/squeezeplay"
+        else:
+            _manufacturer = ""
```

Comment:
Set to `None` if not set

Reply:
OK - was just trying to avoid "undefined" in the device list which is a bit messy.  Is your preference "Undefined" then rather than just a blank?

Reply:
I think `None` is the default?

Reply:
returning None leaves the manufacturer as <unknown>.  I've changed it to None - number of unknown devices is going to be very small anyway.

Reply:
Be beware that device infoes don't overwrite. So try to readd the integration and check again

Reply:
Yes, deleted and re-added.  None returns <unknown>, but that's OK - that'll only be a very small number of devices (Squeezebox, Squeezelite, Squeezeplay will cover the vast majority).